Sources/ButtonBarViewCell.swift

```diff
@@ -33,8 +33,7 @@ open class ButtonBarViewCell: UICollectionViewCell {
         super.init(coder: aDecoder)
         
         isAccessibilityElement = true
-        accessibilityTraits |= UIAccessibilityTraitButton
-        accessibilityTraits |= UIAccessibilityTraitHeader
+        accessibilityTraits = [.button, .header]
```

Comment:
I think you should use the `insert` or `formUnion` function here. The previous logic was adding the two traits to whatever was already in accessibilityTraits.